app/components/additional-provider-list.js

```diff
@@ -28,9 +28,11 @@ export default Component.extend(Analytics, {
         const pairedList = [];
         for (let i = 0; i < sortedList.get('length'); i += 2) {
             const pair = [];
-            pair.pushObject(sortedList.objectAt(i));
+            // provider in pair needs to be a list because that's
+            // what the discover controller is expecting
+            pair.pushObject([sortedList.objectAt(i)]);
```

Comment:
Was this broken before? Just curious why it changed. Different input data now?

Reply:
It was not, I just made the serialization/deserialization consistent. I could add "convert to list if not a list" as part of that if you prefer. I figured it would be more clear if it breaks for improper type.

Reply:
nah, was more just curious